src/payment/strategies/cybersource/cardinal.ts

```diff
@@ -1,9 +1,11 @@
 import Address from '../../../address/address';
 import BillingAddress from '../../../billing/billing-address';
-import { CreditCardInstrument } from '../../payment';
+import { CreditCardInstrument, VaultedInstrument } from '../../payment';
 
 export const CardinalSignatureValidationErrors = [100004, 1010, 1011, 1020];
 
+export type CardinalSupportedPaymentInstrument = CreditCardInstrument | VaultedInstrument;
```

Comment:
This file should only contain types that are specific to the cardinal sdk. `CardinalSupportedPaymentInstrument` is a responsibility of the `cardinal-client`, so the type should probably live there.